domains/kenny

```diff
@@ -0,0 +1 @@
+        "email": "kennybugeja@gmail.com"
```

Comment:
This file is invalid; it doesn't have a json file ending and is also invvalid json so please either fix it / delete the file.

Reply:
i deleted it

Reply:
Doesn't show as deleted on here, can you check again please?